lib/ansible/utils/display.py

```diff
@@ -619,14 +654,21 @@ def do_var_prompt(self, varname, private=True, prompt=None, encrypt=None, confir
             result = wrap_var(result)
         return result
 
-    def _set_column_width(self):
+    def _set_column_width(self) -> None:
         if os.isatty(1):
             tty_size = unpack('HHHH', fcntl.ioctl(1, termios.TIOCGWINSZ, pack('HHHH', 0, 0, 0, 0)))[1]
         else:
             tty_size = 0
         self.columns = max(79, tty_size - 1)
 
-    def prompt_until(self, msg, private=False, seconds=None, interrupt_input=None, complete_input=None):
+    def prompt_until(
+        self,
+        msg: str,
+        private: bool = False,
+        seconds: int | None = None,
+        interrupt_input: t.Iterable[bytes] | None = None,
+        complete_input: t.Iterable[bytes] | None = None,
+    ) -> bytes | None:
```

Comment:
What's the code branch returning `None`? Can `current_worker.worker_queue.get()` return `None`?

Reply:
@mkrizek did you verify this? Because it was just a guess on my side…

Reply:
The original was likely an oversight or "copy&paste error" on my side.

Reply:
Maybe.. But if that invocation may return `None`, then it'd need to be kept.